features/gardener/exec.config

```diff
@@ -5,8 +5,12 @@ set -Eeuo pipefail
 chmod u-s /sbin/mount.nfs /sbin/mount.cifs
 
 # Disable containerd, Gardener will have to enable it
-systemctl disable containerd
-systemctl disable ssh
+# TODO: for the time being keep these here as it's more obvious, but they should be moved to file.include
+mkdir -p /etc/systemd/system-preset
+echo "disable containerd.service" > /etc/systemd/system-preset/00-containerd-disable.preset
```

Comment:
what's the point of using preset instead of directly disable?

Reply:
In this case it's going to have the same effect, it's just to make it more readable.